### PR TITLE
Update csv file name

### PR DESCRIPTION
**Pull request by cdp-finbourne**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch
- [x] Notebook outputs do not contain any priviledged data

# Description of the PR
Update csv file name

### CODE REVIEW

File: examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb [hunk 1]
```diff
@@ -1030,160 +1654,405 @@
...
-     "execution_count": 10,
+     "execution_count": 6,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "holdings = pd.concat(holdings)\n",
-    "holdings.to_csv('holdings_exctract.csv')\n",
+    "holdings.to_csv('holdings_extract.csv')\n",
```
Comment (MikeMcGarry) [inline examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb:2055]: Did you also update the name of the output file that exists next to this notebook?

Comment (cdp-finbourne) [reply examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb]: I've removed the file from the repo

**Comment by cdp-finbourne** [comment]
Please review

**Comment by cdp-finbourne** [comment]
I've removed the index from the output

**Comment by cdp-finbourne** [comment]
Please review - the index has been removed from the output

File: examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb (hunk 1 above)
Comment (MikeMcGarry) [reply examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb]: Good idea.

**Review by MikeMcGarry — APPROVED** [review]
Looks great, we just need to make sure that the tests before merging.

**Pull request merged**